tests/Backend/Workspaces/ExasolWorkspacesUnloadTest.php

```diff
@@ -0,0 +1,238 @@
+<?php
+
+namespace Keboola\Test\Backend\Workspaces;
+
+use Keboola\StorageApi\ClientException;
+use Keboola\Test\Backend\WorkspaceConnectionTrait;
+use Keboola\Test\Backend\Workspaces\Backend\WorkspaceBackendFactory;
+
+class ExasolWorkspacesUnloadTest extends ParallelWorkspacesTestCase
+{
+    use WorkspaceConnectionTrait;
+
+    public function testCreateTableFromWorkspace()
+    {
+        // create workspace and source table in workspace
+        $workspace = $this->initTestWorkspace();
+
+        $tableId = 'Languages3';
+
+        $backend = WorkspaceBackendFactory::createWorkspaceBackend($workspace);
+        $backend->dropTableIfExists($tableId);
+
+        $connection = $workspace['connection'];
+
+        $db = $this->getDbConnection($connection);
+
+        $quotedTableId = $db->getDatabasePlatform()->quoteIdentifier(sprintf(
+            '%s.%s',
+            $connection['schema'],
+            $tableId
+        ));
+
+        $db->query("create table $quotedTableId (
+			[Id] integer not null,
+			[Name] varchar(50) not null
+		);");
+        $db->query("insert into $quotedTableId ([Id], [Name]) values (1, 'cz');");
```

Comment:
tohle je pozustatek ze Synapse ne? Ta trida je identicka se Synapse (krome nazvu)

Reply:
Tady ten test sa kopíruje mezi backendama protože má specifické věci, toto je pozůstatek synapse, ale Exasol taky podporuje escapování pomocí `[]` takže to jede. Ale mělo by sa to fixnů :+1:

Reply:
tomu rozumim, ze to kopirujeme a upravujeme pro dany backend. Jen me to pres tu syntaxi koplo do oci. Ale jestli to jede, tak je to cajk